tests/test-windows.sh

```diff
@@ -33,10 +33,12 @@ sed -i -E 's|HOOK_NAME=.*|HOOK_NAME=\${HOOK_NAME:-\$(basename "\$0")}|' "$ROOT_D
 
 if [ -n "${EXTRA_INSTALL_ARGS}" ]; then
     sed -i "s|sh \"$ROOT_DIR\"/install.sh|sh \"$ROOT_DIR\"/install.sh \${EXTRA_INSTALL_ARGS}|g" /var/lib/tests/step-* || exit 6
+    # shellcheck disable=SC1117
```

Comment:
Why is this suddenly needed?

Reply:
shellcheck on my machine, pre-commit hook complained

Reply:
Can you please check what version of shellcheck you have? Thanks!